lib/ask/runtime/broker.ex

```diff
@@ -164,8 +165,19 @@ defmodule Ask.Runtime.Broker do
   end
 
   defp set_stalled_respondents_as_failed(survey) do
+    from(r in assoc(survey, :respondents), where: r.state == "stalled" and r.disposition == "queued")
```

Comment:
Actually, the disposition shouldn't change in this case. Sending an additional commit to fix this.